File: particle-auth/android/src/main/kotlin/network/particle/flutter/bridge/model/InitData.java
package network.particle.flutter.bridge.model;

import com.google.gson.annotations.SerializedName;


public class InitData {

    @SerializedName("chain_name")
    public String chainName;

    @SerializedName("chain_id")
    public int chainId;

    @SerializedName("chain_id_name")
    public String chainIdName;

    @SerializedName("env")
    public String env;


}

File: particle-auth/android/src/main/kotlin/network/particle/auth_flutter/bridge/model/InitData.kt
package network.particle.auth_flutter.bridge.model

import com.google.gson.annotations.SerializedName

class InitData {
    @SerializedName("chain_name")
    var chainName: String? = null

    @SerializedName("chain_id")
    var chainId: Long = 0

    @SerializedName("chain_id_name")
    var chainIdName: String? = null

    @SerializedName("env")
    var env: String? = null
}